Trappist/src/Promact.Trappist.Web/wwwroot/app/profile/profile-edit/profile-edit.html

```diff
@@ -23,14 +23,14 @@
                 </div>
                 <div class="form-group">
                     <label>Contact Number</label>
-                    <input type="text" class="form-control half-width" name="phonenumber" [(ngModel)]="editUser.phoneNumber" #phonenumber="ngModel" pattern="[0-9]+$" maxlength="20" minlength="10" />
+                    <input type="text" class="form-control half-width" name="phonenumber" [(ngModel)]="editUser.phoneNumber" #phonenumber="ngModel" pattern="^[+]?([0-9]{10,20})$" maxlength="20" />
                     <div class="errors-container">
-                        <span *ngIf='phonenumber.errors && (phonenumber.dirty || phonenumber.touched)' class='error-msg'>Phone Numnber must be have 10 digits and should only contain numbers </span>
+                        <span *ngIf='phonenumber.errors && (phonenumber.dirty || phonenumber.touched)' class='error-msg'>Phone Numnber must have atleast 10 digits with an optional country code</span>
```

Comment:
please give a consistent validation message across the pages. register HTML has a different message.